src/module/actor/character/feats.ts

```diff
@@ -42,15 +43,27 @@ class CharacterFeats extends Collection<FeatCategory> {
         this.createCategory({
             id: "ancestry",
             label: "PF2E.FeatAncestryHeader",
-            featFilter: "ancestry-" + actor.ancestry?.slug,
+            featFilter: actor.system.details.ancestry?.trait
+                ? `ancestry-${actor.system.details.ancestry?.trait}`
+                : null,
```

Comment:
Later on it'd be nice to do versatile heritages, but I guess there's no need to rush that yet.

Reply:
Yeah, I thought about that, but it was a detour while on the detour.